legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/testSuite/dynaFunctions/boolean.pure

```diff
@@ -18,20 +18,6 @@ import meta::pure::test::*;
 
 //operations on booleans
 
-function <<paramTest.Test>> meta::relational::tests::dbSpecificTests::sqlQueryTests::dynaFunctions::sqlTrue::testSqlTrue(config:DbTestConfig[1]):Boolean[1]
```

Comment:
you will need to add them in the ignored dyna fn list in testSuiteCompletenessTests.pure